### ISSUE TITLE
Atom on Windows seems to get confused about update availability

### ISSUE THREAD

**Issue by jolsenmultiad** [atom/atom#11526]
Summary: Sometimes Atom gets confused when there is an Atom update--it indicates an update is available via squirrel icon in status bar, but when I try to update it says no update is available.

Longer version: According to the Help menu, I have 1.7.1 installed. The squirrel icon is displayed in the status bar. If I click the icon, a pane opens and displays a message that "Atom is update to date!" and shows version 1.7.1 at the top of the pane under the Atom logo. Automatically download updates is selected. If I click  "Check Now" an alert appears and says "no update available" and "Version 1.7.1 is the latest version." If I go to the Help menu and select "check for update" I get the same alert. When going to https://atom.io/docs, I see it reports the most recent version is 1.7.2. So, it seems that somehow the status bar knows there is an update available even though the rest of Atom doesn't.

If I restart Atom, it will update to 1.7.2 (this has happened to me quite a few times in the past, pretty much with every update that gets released). Atom will function normally (no squirrel icon) until the next update is released, at which time I expect that I'll see this issue again.

I see an app-1.7.1 and app-1.7.2 in my AppData/Local/atom folder.

OS: Windows 10, version 1511 build 10586.218
atom.exe --version (from app-1.7.1 directory; running atom --version from path returns no output) returns:
  Atom    : 1.7.1
  Electron: 0.36.8
  Chrome  : 47.0.2526.110
  Node    : 5.1.1

apm --version returns:
  apm  1.9.2
  npm  2.13.3
  node 0.10.40
  python 2.7.9
  git 2.7.1.windows.2
  visual studio

**Comment by lee-dohm** [comment]
How did you originally install Atom? Have you tried uninstalling Atom and reinstalling it?

**Comment by Ben3eeE** [comment]
Duplicate of https://github.com/atom/atom/issues/10152?

**Comment by jolsenmultiad** [comment]
I originally installed Atom via the installer. I then tried installing the beta because it had a fix not yet in stable and because it was supposed to run "side by side" with the stable version. It seems like it simply took the place of the stable version. I ended up reinstalling stable and that's what I've ran for a long time. 

Yesterday, I installed Atom 1.8.0-b2 to get a fix not yet on stable, so I don't know if the problem will persist. If it does, I will try to uninstall atom completely and re-install it to see if that resolves the issue.

I looked at #10152 and this seems like it could be a duplicate, along with the other referenced issue #8265. I'm sorry I didn't find those before posting.
